flexget/utils/requests.py

```diff
@@ -231,7 +231,7 @@ def request(self, method, url, *args, **kwargs):
 
         try:
             log.debug('Fetching %s' % url)
-            result = requests.Session.request(self, method, url, *args, **kwargs)
+            result = super(Session, self).request(method, url, *args, **kwargs)
```

Comment:
We should update `__init__` to use super too, and add *args and **kwargs that get passed up the chain if unknown to us.